posthog/models/event/query_event_list.py

```diff
@@ -112,29 +114,75 @@ def query_events_list(
         prop_filters += " AND {}".format(action_query)
         prop_filter_params = {**prop_filter_params, **params}
 
-    # if not using hogql-powered "select" to fetch certain columns, return an array of objects
-    if not isinstance(select, list):
-        order = "DESC" if len(order_by) == 1 and order_by[0] == "-timestamp" else "ASC"
-        if where:
-            raise ValueError("Cannot use 'where' without 'select'")
-        if prop_filters != "":
-            return insight_query_with_columns(
-                SELECT_EVENT_BY_TEAM_AND_CONDITIONS_FILTERS_SQL.format(
-                    conditions=conditions, limit=limit_sql, filters=prop_filters, order=order
-                ),
-                {"team_id": team.pk, "limit": limit, "offset": offset, **condition_params, **prop_filter_params},
-                query_type="events_list",
-                workload=Workload.OFFLINE,
-            )
-        else:
-            return insight_query_with_columns(
-                SELECT_EVENT_BY_TEAM_AND_CONDITIONS_SQL.format(conditions=conditions, limit=limit_sql, order=order),
-                {"team_id": team.pk, "limit": limit, "offset": offset, **condition_params},
-                query_type="events_list",
-                workload=Workload.OFFLINE,
-            )
+    order = "DESC" if len(order_by) == 1 and order_by[0] == "-timestamp" else "ASC"
+    if prop_filters != "":
+        return insight_query_with_columns(
+            SELECT_EVENT_BY_TEAM_AND_CONDITIONS_FILTERS_SQL.format(
+                conditions=conditions, limit=limit_sql, filters=prop_filters, order=order
+            ),
+            {"team_id": team.pk, "limit": limit, "offset": offset, **condition_params, **prop_filter_params},
+            query_type="events_list",
+            workload=Workload.OFFLINE,
+        )
+    else:
+        return insight_query_with_columns(
+            SELECT_EVENT_BY_TEAM_AND_CONDITIONS_SQL.format(conditions=conditions, limit=limit_sql, order=order),
+            {"team_id": team.pk, "limit": limit, "offset": offset, **condition_params},
+            query_type="events_list",
+            workload=Workload.OFFLINE,
+        )
```

Comment:
Looks like these two branches only differ with regard to the SQL query and params. Can we make the
```Python
return insight_query_with_columns(
    sql_query,
    sql_params,
    query_type="events_list",
    workload=Workload.OFFLINE,
)
```
call outside of the branches for readability?

Reply:
Yeah, agreed. This is the old events list code though, and I wouldn't want to refactor it too much. I'd like to replace this entire function with a call to the `v2` query, only returning data in the old format.